lib/modules/manager/buildpacks/extract.ts

```diff
@@ -0,0 +1,103 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { getDep } from '../dockerfile/extract';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { type ProjectDescriptor, ProjectDescriptorToml } from './schema';
+
+const dockerPrefix = regEx(/^docker:\/?\//);
+const dockerRef = regEx(
+  /^((?:[a-z\d](?:[a-z\d-]{0,61}[a-z\d])?(?:\.[a-z\d](?:[a-z\d-]{0,61}[a-z\d])?)*)(?::\d{2,5}\/)?)?[a-z\d]+((\.|_|__|-+)[a-z\d]+)*(\/[a-z\d]+((\.|_|__|-+)[a-z\d]+)*)*(?::(\w[\w.-]{0,127})(?:@sha256:[A-Fa-f\d]{32,})?|@sha256:[A-Fa-f\d]{32,})$/,
+);
+
+function isDockerRef(ref: string): boolean {
+  if (ref.startsWith('docker:/') || dockerRef.test(ref)) {
```

Comment:
```suggestion
  if (ref.startsWith(dockerPrefix) || dockerRef.test(ref)) {
```

or is this somehow wrong?

Reply:
I believe `startsWith` accepts only string as input, while `dockerPrefix` is a regex (to capture both cases, when there's a single and double slashes after `docker:`)